images/onboarding/wcpay.svg

```diff
@@ -0,0 +1,26 @@
+<svg
```

Comment:
Nice 👍  We can probably remove the JS SVG under `payments/images/wcpay.js` in favor of this.  The old payment method will need to use an `<img` tag for the time being.

Reply:
Good call, done!